pkg/proto/nbcontract/v1/networkconfig.proto

```diff
@@ -1,31 +1,38 @@
 syntax = "proto3";
 package nbcontract.v1;
-import "pkg/proto/nbcontract/v1/featurestate.proto";
 
 message NetworkConfig {
-  NetworkPluginType network_plugin = 1;
-  NetworkPolicyType network_policy = 2;
-  NetworkModeType network_mode = 3;
+
+  // NetworkPlugin is the network plugin to be used by the cluster. Options are NONE, AZURE, KUBENET.
+  NetworkPlugin network_plugin = 1;
+
+  // NetworkPolicy is the network policy to be used by the cluster.
+  // This is still needed to compute ENSURE_NO_DUPE_PROMISCUOUS_BRIDGE. 
+  // Other than that, it is not used by others. See the discussions here https://github.com/Azure/AgentBaker/pull/4241#discussion_r1554283228
+  NetworkPolicy network_policy = 2;
+
+  // NetworkMode is the network mode to be used by the cluster. Options are BRIDGE and TRANSPARENT.
+  NetworkMode network_mode = 3;
```

Comment:
I believe the conclusion was that we can remove `network_mode`. It was always be `transparent` so we can just directly pass that to cse_cmd for now until we update the scripts, but it can be removed from contract.

Reply:
Totally! Missed that conclusion. Removed network_mode from contract now